features/command_line/bisect.feature

```diff
@@ -106,8 +107,10 @@ Feature: Bisect
           - ./spec/calculator_7_spec.rb[1:1]
           - ./spec/calculator_8_spec.rb[1:1]
           - ./spec/calculator_9_spec.rb[1:1]
+      Checking that failures are order-dependent..
+       - Running: rspec ./spec/calculator_1_spec.rb[1:1] --seed 1234 (n.nnnn seconds) failure is order-dependent
```

Comment:
Sure, I'll sort something out there (might put it on a separate line):

```
Checking that failures are order-dependent...
 - Running: <snip>
Failure is order-dependent
```

(with the above-mentioned changes to the "is order-dependent" copy, of course)